spec/02-integration/05-proxy/06-ssl_spec.lua

```diff
@@ -212,6 +212,8 @@ for _, strategy in helpers.each_strategy() do
         nginx_http_proxy_ssl_trusted_certificate = "../spec/fixtures/kong_spec.crt",
       })
 
+      ngx.sleep(0.01)
```

Comment:
Why is this one needed?

Reply:
We need some time to wait for event broker/client to establish unix domain socket connect.